bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+from bot.constants import Tokens
+
+log = logging.getLogger(__name__)
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["earth"])
+    async def earth_photos(self, ctx: commands.Context) -> None:
+        """Returns a random photo of earth, sourced from Unsplash."""
+        async with ctx.typing():
+            async with self.bot.http_session.get(
+                    'https://api.unsplash.com/photos/random',
+                    params={"query": "earth", "client_id": Tokens.unsplash_access_key}
+            ) as r:
+                jsondata = await r.json()
+                linksdata = jsondata.get("urls")
+                embedlink = linksdata.get("regular")
+                downloadlinksdata = jsondata.get("links")
+                userdata = jsondata.get("user")
+                username = userdata.get("name")
+                userlinks = userdata.get("links")
+                profile = userlinks.get("html")
+            # Extra request as per Unsplash Guidelines
+            async with self.bot.http_session.get(
+                downloadlinksdata.get("download_location"),
+                    params={"client_id": Tokens.unsplash_access_key}
+            ) as _:
+                pass
+
+            embed = discord.Embed(
+                title="Earth Photo",
+                description="A photo of Earth 🌎  from Unsplash.",
+                color=Colours.grass_green
+            )
+            embed.set_image(url=embedlink)
+            embed.add_field(name="Author", value=f"Photo by [{username}]({profile}) on Unsplash.")
```

Comment:
There is still a bit more that needs to be added. [This article](https://help.unsplash.com/en/articles/2511315-guideline-attribution) recommends that you have both the author and the site links. It also requires you use a referral link instead of a regular link. Check the article for all the specifics.

Reply:
oh, whoops! I'll get going on that